docs/configuration/readme.md

```diff
@@ -127,23 +127,63 @@ Pomerium should _never_ be exposed to the internet without TLS encryption.
 
 :::
 
-### Certificate
+### Autocert
 
-- Environmental Variable: either `CERTIFICATE` or `CERTIFICATE_FILE`
-- Config File Key: `certificate` or `certificate_file`
-- Type: [base64 encoded] `string` or relative file location
-- Required
+- Environmental Variable: `AUTOCERT`
+- Config File Key: `autocert`
+- Type: `bool`
+- Optional
+
+Turning on autocert allows Pomerium to automatically retrieve, manage, and renew public facing TLS certificates from [Let's Encrypt][letsencrypt] for each of your managed pomerium routes as well as for the authenticate service. This setting must be used in conjunction with `Certificate Folder` as Autocert must have a place to persist, and share certificate data between services. Provides [OCSP stapling](https://en.wikipedia.org/wiki/OCSP_stapling).
+
+This setting can be useful in a situation where you do not have Pomerium behind a TLS terminating ingress or proxy that is already handling your public certificates on your behalf.
 
-Certificate is the x509 _public-key_ used to establish secure HTTP and gRPC connections.
+:::warning
+
+By using autocert, you agree to the [Let's Encrypt Subscriber Agreement](https://letsencrypt.org/documents/LE-SA-v1.2-November-15-2017.pdf). There are [_strict_ usage limits](https://letsencrypt.org/docs/rate-limits/) per domain you should be aware of.
+
+:::
+
+:::warning
 
-### Certificate Key
+Autocert requires that port `443` be accessible from the internet in order to complete a [TLS-ALPN-01 challenge](https://letsencrypt.org/docs/challenge-types/#tls-alpn-01).
 
-- Environmental Variable: either `CERTIFICATE_KEY` or `CERTIFICATE_KEY_FILE`
-- Config File Key: `certificate_key` or `certificate_key_file`
+:::
+
+### Certificate Folder
+
+- Environmental Variable: either `CERT_FOLDER`
```

Comment:
See other comment in about parameter name

Reply:
👍